src/lib/DiscordWorker.ts

```diff
@@ -57,14 +57,14 @@ function addCustomMethods(job: DiscordJob) {
   }
 
   job.sendMessage = async (msg: string) => {
-    message = await discord.sendMessage(job.data, msg)
+    message = job.data.threadId ? await discord.sendMessage(job.data.threadId, msg) : null
```

Comment:
send message is called in so many places so i guess we need the workaround here for now :)